tool/doc.dart

```diff
@@ -208,16 +214,20 @@ Future<void> generateDocs(String? dir,
   // Fetch since info.
   await fetchSinceInfo(auth);
 
+  var fixStatusMap = await fetchFixStatusMap();
+  print(fixStatusMap);
```

Comment:
Is the map long? Is this for debugging or should this be included in the submitted PR?

Reply:
Good catch!  That was a stray debugging call.  (Removed.)